Lambda-School-Labs/CS10-quizzer
# Abbott/sendgrid

# Description

Adds SendGrid dependencies and basic test functionality to the server.

## Type of change


# How Has This Been Tested?

- Update your local environment settings file with the SendGrid API key (check CodeShip ENV settings).
- Run `pipenv install` to install the new dependencies.
- Open up `quizzes/api.py` in your favorite code editor, and change line 49 (`to_email = Email('test@example.com')`) to an email address you can access.
- While in your virtual environment (`pipenv shell`) run the server (`python manage.py runserver`) and then navigate to `localhost:8000/sendgrid/` - This will trigger SendGrid to send an email to the address you added `api.py`.
    - Note that this email may be sent to your Spam folder.

# Checklist:
